Framework/API/inc/MantidAPI/HistoWorkspace.h

```diff
@@ -0,0 +1,54 @@
+#ifndef MANTID_API_HISTOWORKSPACE_H_
+#define MANTID_API_HISTOWORKSPACE_H_
+
+#include "MantidAPI/DllConfig.h"
+#include "MantidAPI/MatrixWorkspace.h"
+
+namespace Mantid {
+namespace API {
+
+/** HistoWorkspace is an abstract base class for MatrixWorkspace types that are
```

Comment:
Yes, needed for the factory functions. I tried to explain this better in a slightly extended Doxygen.